components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -78,6 +90,16 @@ export class ProjectsDropdownComponent implements OnInit, OnChanges {
     }
   }
 
+  handleFilterKeyUp(): void {
+    this.filteredProjects = this.filterProjects(this.filterValue);
+  }
+
+  filterProjects(value: string): ProjectChecked[]  {
+    return this.projectsArray.filter(project =>
+      project.id.toLowerCase().indexOf(value.toLowerCase()) > -1
+    );
```

Comment:
Going back to a comment in the other related PR, I think you can (a) reuse the function and (b) make it more clear, by changing to:
```
return this.projectsArray.filter(projectContainsString);
```
...and sharing the `projectContainsString` function.


BTW, the above is shorthand for this:
```
return this.projectsArray.filter(project => projectContainsString(project));
```

Reply:
Love that shorthand!  If it's ok, i'm going to opt to save this portion for when I convert them both to ngrx.  That way I can do them both in the same branch and connect their shared pieces.